Qt.py

```diff
@@ -204,18 +225,18 @@ def _init():
     this has executed.
 
     """
-
+    
     preferred = os.getenv("QT_PREFERRED_BINDING")
     verbose = os.getenv("QT_VERBOSE") is not None
     bindings = (_pyside2, _pyqt5, _pyside, _pyqt4)
 
     if preferred:
-
+        
```

Comment:
Have a look and see if you can't remove these whitespaces inbetween lines. A linter could help here.